mesonbuild/scripts/depscan.py

```diff
@@ -15,12 +15,23 @@
 import pathlib
 import pickle
 import re
+import os
+import sys
 import typing as T
 
 from ..backend.ninjabackend import TargetDependencyScannerInfo
+from ..compilers.compilers import lang_suffixes
 
-import_re = re.compile('\w*import ([a-zA-Z0-9]+);')
-export_re = re.compile('\w*export module ([a-zA-Z0-9]+);')
+cpp_import_re = re.compile('\w*import ([a-zA-Z0-9]+);')
+cpp_export_re = re.compile('\w*export module ([a-zA-Z0-9]+);')
+
+FORTRAN_INCLUDE_PAT = r"^\s*#?include\s*['\"](\w+\.\w+)['\"]"
```

Comment:
This could be in a separate PR, but to speed this up, the `#include` syntax isn't used by any compiler I've found. I added this when I was overhauling the regex for Fortran, but since realized no current compiler allows this syntax. 

Suggest:
```python
FORTRAN_INCLUDE_PAT = r"^\s*include\s*['\"](\w+\.\w+)['\"]"
```